client/src/main/java/de/rebleyama/client/RebleyamaClient.java

```diff
@@ -92,7 +92,7 @@ public boolean keyUp(int keycode) {
 /**
  * Stub method for recognizing keypress
  * This triggers when any key is typed (pressed and released)
- * @param keycode keycode of the key that was typed
+ * @param character character that was typed
```

Comment:
Why are you changing this? Are you sure this still works?

Reply:
Alright then. This was taken from a tutorial on the libGDX git repo, so not sure why they would have that if it doesn't work...